File: T5/T5_ExamenBD/app/src/main/java/com/system/develop/t5_examenbd/utils/Alumno.java
package com.system.develop.t5_examenbd.utils;

public class Alumno {

    String nombre, apellido, curso;

    public Alumno(String nombre, String apellido, String curso) {
        this.nombre = nombre;
        this.apellido = apellido;
        this.curso = curso;
    }

    public Alumno(String nombre, String apellido) {
        this.nombre = nombre;
        this.apellido = apellido;
    }

    public String getNombre() {
        return nombre;
    }

    public String getApellido() {
        return apellido;
    }

    public String getCurso() {
        return curso;
    }
}

File: Kotlin/T1/codigos_ret/Objetos/src/model/Alumno.kt
package model

import kotlin.time.Duration.Companion.minutes

class Alumno(
    nombre: String,
    apellido: String,
    correo: String,
    telefono: Int,
    var curso: String,
    var nivel: Int,
    var sueldo: Double?
) :
    Usuario(nombre, apellido, correo, telefono) {

    init {
        this.nombre = nombre+"_alumno"
    }

    fun trabajarConSueldo(){
        var division:Double? = sueldo?.div(12.0)?:0.0
        var muliplicacion:Double = (sueldo?.times(12.0)?:0.0)/12
        // var salario = sueldo.minus()
    }
    override fun mostrarDatos() {
        super.mostrarDatos()
        println("Curso $curso")
        println("Nivel $nivel")
    }

    fun asistirClase(){
        println("Estoy asistiendo a una clase")
    }

    override fun obtenerInformacion() {
        println("La informacion del alumno es:")
        println("$curso")
    }

}